### DISCUSSION TITLE
Create a migration via the CLI `migration:create` without schema diff

### DISCUSSION THREAD

**Discussion by pantajoe** [mikro-orm/mikro-orm#4333] [Q&A] [answered]
Hi there! 👋🏽 

Is it currently possible to create a migration via the CLI using `mikro-orm migration:create` if there is no schema diff present?

For me, the use case is to create/modify data or change the structure of data in the database without having changed anything in my entities.

I.e., I would like to create a migration like this via the CLI without having to manually create the file in my `migrations` directory and find out the current timestamp to paste into the filename.

```ts
class Migration123455789_AddDataToTable {
  async up() {
    const em = createEntityManager(this.ctx)
    new SampleFactory(em).run()
    await em.flush()
  }
  
  async down() {
    this.addSql(`delete from "sample" where some_condition = true;`)
  }
}
```

**Comment by B4nan** [comment, marked as answer]
Yes, there is `--blank` flag.

```bash
npx mikro-orm migration:create --blank
```

**Reply by pantajoe** [reply]
Works like a charm, thanks a lot!
